sample-app/src/pages/StandardLayout.tsx

```diff
@@ -26,7 +26,7 @@ const StandardLayout: LayoutComponent = ({ page }) => {
       {isVertical
         ? <SearchBar
           placeholder='Search...'
-          isVertical={true}
+          isVertical={isVertical}
```

Comment:
I was thinking originally that if this was java intellij would yell at me for using a variable for a value that was always true (here isVertical is always true since it's guarded by the ternary operator) but on second thought it's probably better to use the variable to make copy pasting less error prone